src/services/objections.session.service.ts

```diff
@@ -12,3 +14,13 @@ export const addToObjectionsSession = (session: Session, key: string, value: any
 export const getValueFromObjectionsSession = (session: Session, key: string) => {
     return session.data[OBJECTIONS_SESSION][key];
 };
+
+export const getValidToken = (session: Session): string | undefined => {
```

Comment:
sorry, should have spotted it earlier, could you rename to getValidAccessToken? makes it clearer as to what the token is for and matches up with the messages being logged.

Reply:
Updated the name